### ISSUE TITLE
Harmonise names between tree view and tabs in Single Page

### ISSUE THREAD

**Issue by floradanna** [seek4science/seek#1133]
**Describe the bug**
1. Recently, names of tabs have been changed as requested in issue #1094. However, the names of the corresponding items in tree view have not been changed accordingly.
2. Experiment overview is shown in tree view (there called "Study table") only for Study and not for Assay.

**To Reproduce**
See screenshot below.

**Expected behavior**
1. Names of tabs and corresponding items in tree view must be the same, since they bring the user to the exact same page/content.
2. Do not show "Study table" in tree view, so that there is consistency with Assays.

**Screenshots**
![Screenshot 2022-07-28 at 12 34 34](https://user-images.githubusercontent.com/51486716/181487381-fae1f110-a792-49a5-8b36-b4153dbb8b50.png)

**Comment by floradanna** [comment]
Describe the bug
In Study, there is a one-to-one correspondency between sub-tabs of Study design tab and the items shown under Study in the tree view (see first screenshot). In Assay, there is no such one-to-one correspondency instead (see second screenshot).

To Reproduce
See screenshots below.

Expected behavior
Implement the one-to-one correspondency between sub-tabs of Assay design tab and the items shown under Assay in the tree view (as in Study), by introducing the Samples table under Assay in the tree view.

First screenshot (Study)
<img width="641" alt="Screenshot 2022-07-29 at 16 18 59" src="https://user-images.githubusercontent.com/51486716/181780038-3b549d0c-c467-4ddc-9b40-02dae9d9f0d5.png">

Second screenshot (Assay)
<img width="502" alt="Screenshot 2022-07-29 at 16 19 16" src="https://user-images.githubusercontent.com/51486716/181780081-d55dfb1e-b590-4653-8755-8d06fd3ff945.png">

**Comment by vdkkia** [comment]
IMO it's weird to open the first 3 tabs by clicking on items in the Tree View while there are 4 tabs.
@floradanna Isn't adding Samples and Table links (with altered names of course) in the Tree View a better option?

**Comment by floradanna** [comment]
I agree. "Experiment overview" should be also shown in the tree view, with in Study and Assay. I just don't know if tree view looks too clutter then, but we should try to know it.

I cannot speak for Rafael, but it makes sense to me.
